broker-daemon/broker-rpc/orderbook-service/get-market-stats.js

```diff
@@ -20,7 +20,7 @@ const ONE_DAY_IN_NANOSECONDS = Big('86400000000000')
  * @param {Logger} request.logger
  * @param {Map<Orderbook>} request.orderbooks
  * @param {Object} responses
- * @param {function} responses.GetMarketStatsResponse
+ * @param {Function} responses.GetMarketStatsResponse
```

Comment:
is a function not a primitive for jsdoc?

Reply:
this should enforce `function`. im not sure why it isn't updating these keys but will take a look

Reply:
I had to look this up: https://github.com/gajus/eslint-plugin-jsdoc/blob/master/.README/rules/check-types.md

We use `eslint-plugin-jsdoc`